llvm/test/tools/spir-to-ir/spir-to-ir.ll

```diff
@@ -0,0 +1,49 @@
+; Check for passthrough abilities
+; RUN: llvm-as %s -o %t.bc
+; RUN: spir-to-ir %t.bc -o %t_1.bc
+; RUN: llvm-dis %t_1.bc -o %t_1.ll
+; RUN: FileCheck %s --input-file %t_1.ll
+
+; Check for SPIR-V conversion
+; RUN: llvm-spirv %t.bc -o %t.spv
```

Comment:
I'm a little bit concerned about direct `llvm-spirv` usage in `llvm/test` (although maybe translator maintainers wouldn't share these concerns). It gets me thinking that perhaps there might be cases when `llvm-lit` is not able to find `llvm-spirv` in the local build directory, but the built wrapper tool can?

In case these concerns are anything more than a mirage, an alternative could be to add a separate `.spt`-formatted text file re-using the same checks. A way to convert SPIR-V between the textual/binary form would be the `-to-binary`/`-to-text` CLI options of the translator.

Reply:
So if we have an `spt` formatted text file and we want to create the `spv` equivalent, wouldn't we still need to use `llvm-spirv`?  I guess I don't understand how we get away from using `llvm-spirv` unless we keep separate `spv` input files to use.

Reply:
Ah, that's a good point... OK, I guess I'm just seeing ghosts with this one.